src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -241,11 +241,13 @@ public AEPartLocation addPart( ItemStack is, final AEPartLocation side, final @N
 								{
 									try
 									{
-										new GridConnection( cn, sn, AEPartLocation.INTERNAL );
+										GridConnection.create( cn, sn, AEPartLocation.INTERNAL );
 									}
 									catch( final FailedConnectionException e )
 									{
 										// ekk!
+										e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1148)